apps/server/jest.config.json

```diff
@@ -1,4 +1,6 @@
 {
     "preset": "ts-jest",
-    "testEnvironment": "node"
+    "testEnvironment": "node",
+    "forceExit": true,
+    "maxWorkers": 1
```

Comment:
Max workers config is fine for now. But keep in mind that you may have to increase it a bit as your test files grow in number.